app/models/grda_warehouse/tasks/service_history/enrollment.rb

```diff
@@ -430,8 +430,8 @@ def default_day
         project_name: project.ProjectName,
         organization_id: project.OrganizationID,
         record_type: nil,
-        housing_status_at_entry: self.LivingSituation,
-        housing_status_at_exit: exit&.HousingAssessment,
+        housing_status_at_entry: self.LivingSituation, # NOTE: we should remove this and rely on the join
+        housing_status_at_exit: exit&.HousingAssessment, # NOTE: we should remove this and rely on the join
```

Comment:
Just flagging

Reply:
Yeah, I didn't want to pull out the assignment yet, but I _believe_ they are no longer in use.